Markdown/tests/syntax_test_markdown.md

```diff
@@ -7867,3 +7867,63 @@ This is a [[wiki link]].
 |         ^^^^^^^^^^^^^ meta.link.reference.wiki.description.markdown
 |         ^^ punctuation.definition.link.begin.markdown
 |                    ^^ punctuation.definition.link.end.markdown
+
+
+# TEST: LATEX EQUATIONS #######################################################
```

Comment:
Please add a test for an unclosed inline math expression that is not highlighted.

And one for the requirement of no whitespace on the inner boundaries of the enclosing punctuation, i.e. `$ 1 + 3 $`.